bundles/org.openhab.core.id/src/main/java/org/openhab/core/id/internal/UUIDResource.java

```diff
@@ -49,7 +49,7 @@
 @JaxrsName(UUIDResource.PATH_UUID)
 @JaxrsApplicationSelect("(" + JaxrsWhiteboardConstants.JAX_RS_NAME + "=" + RESTConstants.JAX_RS_NAME + ")")
 @Path(UUIDResource.PATH_UUID)
-@RolesAllowed({ Role.ADMIN })
+@RolesAllowed({ Role.ADMIN, Role.USER })
 @SecurityRequirement(name = "oauth2", scopes = { "admin" })
```

Comment:
Do we have to remove the `SecurityRequirement` to make this endpoint accessible for users? I could not find any other user defined interface method with it.

Reply:
Good point. I actually didn't notice that we have those annotations and am not sure how/if they are considered at all. But it seems to make sense to remove it indeed.

Reply:
Thanks for removing it. Can you please apply the formatter?

Reply:
Sorry, didn't notice. Done now!